src/leader.py

```diff
@@ -31,6 +32,16 @@ class Leader:
 
     STRAIGHT_ANGLE = 90
     """The angle that the hardware associates as straight."""
+    DEMO_SPEED = 40
+    """The speed the leader vehicle drives at."""
+    DEMO_STRAIGHT_TIME = 10
```

Comment:
We're going to be using these functions at Design Day. The cars can travel pretty far in 10 seconds. We're not going to have that much space.